src/lightning_app/cli/commands/connection.py

```diff
@@ -16,14 +19,14 @@
 from lightning_app.utilities.network import LightningClient
 
 _HOME = os.path.expanduser("~")
-_PPID = str(psutil.Process(os.getpid()).ppid())
+_PPID = os.getenv("LIGHTNING_CONNECT_PPID", str(psutil.Process(os.getpid()).ppid()))
 _LIGHTNING_CONNECTION = os.path.join(_HOME, ".lightning", "lightning_connection")
 _LIGHTNING_CONNECTION_FOLDER = os.path.join(_LIGHTNING_CONNECTION, _PPID)
+_PROGRESS_TOTAL = 60
```

Comment:
We may consider to use an indeterminate progress bar instead of faking a time https://rich.readthedocs.io/en/stable/progress.html#indeterminate-progress

Reply:
I would prefer to keep the current one for now. It feels quite natural and adapts to different installation time.
Let's revisit if needed.